nursery/query-remote-server-for-availible-data.yml

```diff
@@ -0,0 +1,10 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: query remote server for availible data
```

Comment:
```suggestion
    name: query remote server for available data
```

+rename file

Reply:
i wonder if there's a more human-readable name for this, like what is the context in which the server "has available data"?